modules/Behaviour/behaviour_manage_edit.php

```diff
@@ -100,18 +97,42 @@
                 $form->addHiddenValue('address', "/modules/Behaviour/behaviour_manage_add.php");
                 $form->addRow()->addClass('hidden')->addHeading('Step 1', __('Step 1'));
 
+
+                //To show other students involved in the incident
+                if(!empty($values['gibbonMultiIncidentID'])) {
+
+                    $resultList = $container->get(BehaviourGateway::class)->selectMultipleStudentsOfOneIncident($values['gibbonMultiIncidentID']);
```

Comment:
When a gateway is used multiple times in a script, it can be more efficient to add a `$behaviourGateway = $container->get(BehaviourGateway::class);` at the top of the script, and then reuse the gateway with that variable.